docs/getting-started/github-apps/migrating-tools-to-github-apps.rst

```diff
@@ -57,7 +57,7 @@ The information above was filled out using the following:
 The ``name`` Field
 ~~~~~~~~~~~~~~~~~~
 
-If you have more than one tool in the same repository, the ``name`` field must be filled out for at least one of the tools to ensure that the tool paths are unique.
+If you have more than one tool in the same repository, the ``name`` field must be filled out for each individual tool's section in your .dockstore.yml file. This is to ensure that the tool paths are unique -- as mentioned in the table above, tool paths are based upon the GitHub repository, so tools without a name in the same repository would have the same path, which we want to avoid. (Strictly speaking, you can leave one of the tools' name fields blank if you would like, but make sure that all other tools in your repository have a name.)
```

Comment:
Nit-picking this to death, but I still don't like "...must be filled out for each individual tool's section" when it's not so, even though you have the caveat in the next sentence. But it's hard to word and it probably is a best practice if you have multiple tools or workflows, so...